diff --git a/node_modules/react-native-circular-slider/src/.ClockFace.js.swp b/node_modules/react-native-circular-slider/src/.ClockFace.js.swp
new file mode 100644
index 0000000..4495eb6
Binary files /dev/null and b/node_modules/react-native-circular-slider/src/.ClockFace.js.swp differ
diff --git a/node_modules/react-native-circular-slider/src/CircularSlider.js b/node_modules/react-native-circular-slider/src/CircularSlider.js
index 32403b4..fc04bba 100644
--- a/node_modules/react-native-circular-slider/src/CircularSlider.js
+++ b/node_modules/react-native-circular-slider/src/CircularSlider.js
@@ -1,4 +1,5 @@
-import React, { PureComponent, PropTypes } from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { PanResponder, View } from 'react-native';
 import Svg, { Circle, G, LinearGradient, Path, Defs, Stop } from 'react-native-svg';
 import range from 'lodash.range';
@@ -45,7 +46,7 @@ function getGradientId(index) {
   return `gradient${index}`;
 }
 
-export default class CircularSlider extends PureComponent {
+export default class CircularSlider extends Component {
 
   static propTypes = {
     onUpdate: PropTypes.func.isRequired,
diff --git a/node_modules/react-native-circular-slider/src/ClockFace.js b/node_modules/react-native-circular-slider/src/ClockFace.js
index c8d0db9..f875dc3 100644
--- a/node_modules/react-native-circular-slider/src/ClockFace.js
+++ b/node_modules/react-native-circular-slider/src/ClockFace.js
@@ -1,4 +1,5 @@
-import React, { PureComponent, PropTypes } from 'react';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
 import { G, Circle, Text, Line } from 'react-native-svg';
 import range from 'lodash.range';
 
diff --git a/node_modules/react-native-circular-slider/src/TimerText.js b/node_modules/react-native-circular-slider/src/TimerText.js
new file mode 100644
index 0000000..dce6b2e
--- /dev/null
+++ b/node_modules/react-native-circular-slider/src/TimerText.js
@@ -0,0 +1,44 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+function calculateTime(totalMinutes) {
+  const minutes = totalMinutes % 60;
+  const hours = (totalMinutes - minutes) / 60;
+
+  return { hours, minutes };
+}
+
+export default class TimerText extends PureComponent {
+
+  static propTypes = {
+    minutesLong: PropTypes.number.isRequired,
+    style: PropTypes.object,
+  }
+
+  render() {
+    const { hours, minutes } = calculateTime(this.props.minutesLong);
+
+    return (
+      <View style={StyleSheet.flatten([styles.container, this.props.style])}>
+        <Text style={styles.text}>{hours}h {minutes}m</Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    color: '#9d9d9d',
+    fontSize: 20,
+    fontWeight: '300',
+  }
+}); 
\ No newline at end of file
